container/app_metadata.yaml

```diff
@@ -0,0 +1,20 @@
+---
+- name: vouch-keystone
+  command: /usr/local/bin/supervisord --config /etc/supervisord-keystone.conf
```

Comment:
Could you make this an array of strings instead. It'll make it easier to handle cases like arguments containing spaces. It's also the standard used by k8s. Same comment for vouch-noauth

Reply:
Thanks - updated the change with commands as a list of strings.